platform.txt

```diff
@@ -8,6 +8,12 @@
 name=Arduino AVR Boards
 version=1.8.3
 
+# pluggable discovery support
+# For more info:
+# https://arduino.github.io/arduino-cli/0.19/platform-specification/#pluggable-discovery
```

Comment:
```suggestion
# https://arduino.github.io/arduino-cli/latest/platform-specification/#pluggable-discovery
```
I think this is better so it will always point to the latest stable version of the docs.